docker/package/model.py

```diff
@@ -494,7 +494,7 @@ class TezosBakingServicesPackage(AbstractPackage):
     # native releases, so we append an extra letter to the version of
     # the package.
     # This should be reset to "" whenever the native version is bumped.
-    letter_version = ""
+    letter_version = "a"
```

Comment:
It would be nice to have the usual `Problem`/`Solution` in this last commit message, but other than that this LGTM now